hs_composite_resource/models.py

```diff
@@ -265,6 +265,18 @@ def delete_old_xml_files(folder=''):
                 # action is needed
                 pass
 
+    def is_aggregation_xml_file(self, file_path):
```

Comment:
It is not just checking the file name rather the full file path as in iRODS. I am not sure parsing the file as XML adds to validation of the file. However, I can add this parsing. This would require copying the file to temp directory in Django for reading.